style.css

```diff
@@ -0,0 +1,127 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Montserrat", sans-serif;
+  font-family: "Poppins", sans-serif;
+}
+
+:root {
+  --main-color: #be3144;
+  --main-text-color: #eee;
+}
+
+.nav {
+  display: flex;
+  justify-content: end;
+  background-color: var(--main-color);
+}
+
+.list {
+  display: flex;
+  list-style: none;
+  margin: 20px;
+}
+
+/* .list p {
+
+  margin: 30px;
+} */
+
```

Comment:
Please don't leave unused code as a comment in your codebase. Unused code as comments makes your code clean.